graphics/xfractint/files/patch-prefix-x11-dirs.diff

```diff
@@ -1,11 +1,11 @@
 --- xfractint-20.04p14/Makefile.orig	2014-05-03 08:12:33.000000000 -0500
-+++ xfractint-20.04p14/Makefile	2019-03-05 16:22:26.000000000 -0600
++++ xfractint-20.04p14/Makefile	2019-03-07 09:49:54.000000000 -0600
 @@ -20,26 +20,21 @@
  
  # Use Xft/fontconfig libraries
  WITHXFT = -DWITH_XFT
 -XFTHFD = /usr/include/freetype2
-+XFTHFD = /opt/X11/include/freetype2
++XFTHFD = /opt/local/include/freetype2
```

Comment:
You cannot assume that $PREFIX is opt/local. You have to put in something like this:

`XFTHFD = __PREFIX__/include/freetype2`

And then do a replace of `__PREFIX__` with `$PREFIX` in the `post-patch` phase.

Reply:
done